client/me/account/index.jsx

```diff
@@ -346,11 +360,11 @@ module.exports = React.createClass( {
 				<FormFieldset>
 					<FormLabel htmlFor="email">{ this.translate( 'Email Address' ) }</FormLabel>
 					<FormTextInput
-						disabled={ this.getDisabledState() }
+						disabled={ this.getDisabledState() || this.hasPendingEmailChange() }
 						id="email"
 						name="email"
 						onFocus={ this.recordFocusEvent( 'Email Address Field' ) }
-						valueLink={ this.valueLink( 'user_email' ) } />
+						valueLink={ this.renderEmailValueLink() } />
```

Comment:
The only issue I have with using `valueLink` for `new_user_email` is that since we don't allow the user to modify `new_user_email` directly, we don't really need two-way binding.

An alternative, while more verbose, would be:

```
<FormTextInput
	disabled={ this.getDisabledState() || this.hasPendingEmailChange() }
	id="email"
	name="email"
	onFocus={ this.recordFocusEvent( 'Email Address Field' ) }
	value={ this.hasPendingEmailChange() ? this.props.userSettings.getSetting( 'new_user_email' ) : null }
	valueLink={ this.hasPendingEmailChange() ? null : this.valueLink( 'user_email' ) } />
```

I just tested that solution and it seemed to work. I don't feel super strongly about this though, so  ¯\_(ツ)_/¯